deploy/libproject/deploy.rb

```diff
@@ -309,6 +309,66 @@ def deploy(cmd_name, args)
   :fn => ->(*args) { deploy("deploy", args) }
 })
 
+def deploy_tanagra(cmd_name, args)
```

Comment:
This is the test deployment command that deploy both API and UI to test environment with single command